bridge_adaptivity/module/forms.py

```diff
@@ -30,7 +30,10 @@ class Meta:
         model = Activity
         exclude = ['collection', 'points']
         widgets = {
-            'stype': forms.HiddenInput(), 'points': forms.HiddenInput(), 'lti_lms_platform': forms.HiddenInput()
+            'stype': forms.HiddenInput(),
+            'points': forms.HiddenInput(),
```

Comment:
`points` are excluded from the form do we need to add hiddenInput for this field?

Reply:
You are right. I deleted this field.